lib/application.js

```diff
@@ -57,6 +57,9 @@ class Application extends api.events.EventEmitter {
     const after = () => {
       if (--count === 0) started();
     };
+    this.frontend.on('broadcastToClients', (...args) => {
+      impress.server.instance.broadcast(...args);
```

Comment:
Shouldn't there be a check that validates that `impress.server.instance` is a `jstp.Server` here?
Since `http.Server` doesn't have the `broadcast()` method, won't this fail with `TypeError: impress.server.instance.broadcast is not a function` when used with a worker serving HTTP?